Models/Website/User.php

```
@@ -388,33 +386,36 @@ public function getAvatar(): string
     /**
      * Method returning link of avatar image that should be displayed.
-     * By defualt, the avatar set manually by the user in their account settings is used. If none has been set, the one
-     * fetched from Discord (downloaded during account creation via API) is returned (if such avatar exists).
-     * If none of the avatars mentioned above is present, the default (empty) one is returned.
+     * If the user has a manually uploaded or Discord-synced avatar, it is served from the avatars folder.
+     * Otherwise, the default avatar is returned.
      * @param bool $appendRandom Should a random number in range 0–31 be appended to the file name, to prevent caching?
      * @return string Link of the profile picture (a random number is appended to the end to prevent caching if
      * the avatar isn't the default one) and the argument $appendRandom is set to TRUE.
      */
     public function getAvatarLink(bool $cacheBust = true): string
```

PHPStan (2.1.54)

[error] PHPDoc tag `@param` references unknown parameter: $appendRandom


(parameter.notFound)